app/services/post_scraper.rb

```diff
@@ -152,11 +152,12 @@ def import_post_from_doc(doc)
     content = doc.at_css('.entry-content').inner_html
 
     @post = Post.new
+    @written = @post.replies.new
```

Comment:
Do we need `@post.written = @written = @post.replies.new` here? I think we do that in the controller.

Reply:
I have no idea

Reply:
we do apparently